src/views/tips/tips.jsx

```diff
@@ -0,0 +1,196 @@
+var React = require('react');
+var injectIntl = require('react-intl').injectIntl;
+var FormattedHTMLMessage = require('react-intl').FormattedHTMLMessage;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var render = require('../../lib/render.jsx');
+
+var MasonryGrid = require('../../components/masonrygrid/masonrygrid.jsx');
+var Page = require('../../components/page/www/page.jsx');
+var Button = require('../../components/forms/button.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+var FlexRow = require('../../components/flex-row/flex-row.jsx');
+var TTTTile = require('../../components/ttt-tile/ttt-tile.jsx');
+var TTTModal = require('../../components/modal/ttt/modal.jsx');
+var Tiles = require('./ttt.json');
+
+require('./tips.scss');
+
+var Tips = injectIntl(React.createClass({
+    type: 'Tips',
+    getInitialState: function () {
+        return {
+            currentTile: Tiles[0],
+            TTTModalOpen: false
+        };
+    },
+    showTTTModal: function (tile) {
+        // expects translated tile
+        this.setState({currentTile: tile});
+        this.setState({TTTModalOpen: true});
```

Comment:
This is minor, but unless you have a specific reason for needing to do this in two `setState` calls, I think it is better to combine them into a single `this.setState({currentTile: tile, TTTModalOpen: true});`